src/physlr-molecules.cc

```diff
@@ -74,6 +79,11 @@ using vertexSet_t = std::unordered_set<vertex_t>;
 using componentToVertexSet_t = std::vector<vertexSet_t>;
 using vertexToComponent_t = std::unordered_map<vertex_t, size_t>;
 using vecVertexToComponent_t = std::vector<vertexToComponent_t>;
+using vertexToIndex_t = std::unordered_map<vertex_t, size_t>; // wanna improve this? checkout boost::bimap
```

Comment:
I would recommend uint64_t instead of size_t for everything

Reply:
So should I change every and each occurrence of `size_t`? Is it fine if I do that in a separate PR?